pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -808,6 +809,9 @@ type PushoverConfig struct {
 	// A title for supplementary URL, otherwise just the URL is shown
 	// +optional
 	URLTitle string `json:"urlTitle,omitempty"`
+	// The time to live definition for the alert notification
+	// +optional
+	TTL *model.Duration `json:"ttl,omitempty"`
```

Comment:
you want v1.Duration (though I see that the pushover duration is a custom type in Alertmanager code, not sure why).

Reply:
> I see that the pushover duration is a custom type in Alertmanager code

sorry, where? any pointer

Reply:
I was referring to https://github.com/prometheus/alertmanager/blob/91a94f00f987048fa4f42856165d49370a3af568/config/notifiers.go#L705

But I notice that we're not 100% compliant with the other Pushover duration fields. We can fix them later but for `ttl` it would be better to use the existing `v1.Duration` type.